app/controllers/admin/users_controller.rb

```diff
@@ -216,17 +221,19 @@ def send_email
     rescue ActionController::ParameterMissing
       respond_to do |format|
         format.json do
-          render json: { error: "Both subject and body are required!" },
+          render json: { error: I18n.t("admin.users_controller.parameter_missing") },
                  content_type: "application/json",
                  status: :unprocessable_entity
         end
       end
     end
 
+    # NOTE: [@rhymes] This should be eventually moved in Admin::Users::Tools::EmailsController
+    # once the HTML response isn't required anymore
```

Comment:
**Quick suggestion:** We no longer need this comment, as `Admin::Users::Tools::EmailsController` no longer exists - let's remove it!:
```suggestion
```